python/ray/scripts/scripts.py

```diff
@@ -647,8 +647,20 @@ def rsync_down(cluster_config_file, source, target, cluster_name):
     required=False,
     type=str,
     help="Override the configured cluster name.")
-def rsync_up(cluster_config_file, source, target, cluster_name):
-    rsync(cluster_config_file, source, target, cluster_name, down=False)
+@click.option(
+    "--sync-workers",
```

Comment:
`--all-nodes`? or `--all` or `--full`?

``--sync-workers`` seems to be specific to workers.

Reply:
`--all-nodes` sounds good. how about `-a` as the shorthand?

Reply:
sounds good!

Reply:
`-A` also works.